.gitignore

```diff
@@ -1,6 +1,7 @@
 /target/
 .direnv
 .envrc
+.idea
```

Comment:
I think this is the first pattern that's editor-specific. I think we can just draw the line after the 5 most popular editors or something (maybe not adding `*.bak` or whatever Notepad.exe uses, for example). I think it makes sense to have this here since it helps significant number of people.

Reply:
I would put it at the end, in a section marked `# Editor-specific ignores`. Maybe also change it to `.idea  # IntelliJ IDEs

Reply:
@ilyagr Sorry, PR was auto-merged, I'll send a separate one with this.